OpenSim/Common/Exception.h

```diff
@@ -67,14 +91,29 @@ class OSIMCOMMON_API Exception  : public std::exception {
     std::string _file;
     /** Line number at which the error occurred. */
     int _line;
+    /** Delimiter/separator used to separate different parts of the message.  */
+    static const std::string _msgSeparator;
 
 //=============================================================================
 // METHODS
 //=============================================================================
 public:
     // CONSTRUCTORS
-    Exception(const std::string &aMsg="",const std::string &aFile="",int aLine=-1);
+    Exception(const std::string &aMsg="",
```

Comment:
I have added the comment.
The old constructor is implemented to do something else. Delegating to the new constructor will change the behavior. I would leave the old constructor as is. It will be removed later.